common/core/web/tools/sentry-manager/src/build.sh

```diff
@@ -0,0 +1,66 @@
+#! /bin/bash
+# 
+# Compiles development-related KeymanWeb resources for use with developing/running tests.
+#   - the Recorder module (for engine tests)
+#   - the DOM module (for touch-alias and element-interface tests)
+
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(greadlink -f "${BASH_SOURCE[0]}" 2>/dev/null || readlink -f "${BASH_SOURCE[0]}")"
```

Comment:
I'd still like to keep this the same as all other "STANDARD BUILD SCRIPT INCLUDES" so maintenance is less fraught.